src/service.ts

```diff
@@ -172,9 +172,9 @@ export function setupService(
     ];
     const cwd = cfg.cwd ? { cwd: cfg.cwd } : {};
     const env = cfg.extraEnv
-      ? { env: Object.assign({}, process.env, cfg.extraEnv) }
-      : {};
-    const options = Object.assign({ stdio }, cwd, env);
+      ? Object.assign({}, process.env, cfg.extraEnv)
+      : process.env;
```

Comment:
:+1:

Reply:
This shouldn't be necessary but in practice it is necessary on windows -- in the nodejs docs it says that lack of `env` option means that `process.env` is used -- wrong.